docs/deploy_streamlit_app.md

```diff
@@ -84,11 +84,26 @@ Streamlit sharing is completely free and is meant to get you started with sharin
 
 ### Access
 
-- Apps are always visible to the entire world.
+- Apps are visible to the entire world.
 - You can only deploy apps that are in a public GitHub repo.
 - Your source code must live in Github. We're looking to expand to other Git hosts soon.
 - Everyone with push access to your repo is automatically a maintainer of the app.
 
+### Long Term
+
+For apps that have been deployed for a long time but are not getting any views or traffic, they will automatically go to sleep after 7 days. This is done to alleviate resources and allow the best communal use of the platform! Here are some need to know's about how this works:
+
+- As the app developer, you will receive an email after 5 days of no traffic on your app.
+- If you would like to keep your app awake, you have one of two choices:
+  - You or a friend can visit the app (create traffic).
```

Comment:
"You or a friend" seems too cute here...

Reply:
No problem, just removed that part entirely so it starts with "Visit ..."